src/odemis/model/test/futures_test.py

```diff
@@ -260,44 +266,65 @@ def testCancelPostEnd(self):
 
         self.assertEqual(self.cancelled, 0)
 
-    def testProgressiveFuture(self):
+    # TODO create own class for progressive future
```

Comment:
Why not just do that directly?

Reply:
You would not have seen what I changed and what was already there :) I will do it now with the update.